src/libraries/Common/src/Interop/Windows/Kernel32/Interop.ReadConsoleInput.cs

```diff
@@ -16,7 +16,7 @@ internal struct KeyEventRecord
         internal short repeatCount;
         internal short virtualKeyCode;
         internal short virtualScanCode;
-        internal ushort uChar; // Union between WCHAR and ASCII char
+        internal char uChar; // Union between WCHAR and ASCII char
```

Comment:
I think it would look better to keep it as `ushort` and cast it as necessary. It avoids any ambiguity for interop marshaling. @elinor-fung @jkoritzinsky What do you think?

Reply:
Agreed - would prefer to keep it as `ushort` for interop and cast as needed.